@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/CfgUserActions.hpp

```diff
@@ -32,7 +32,6 @@ class CfgUserActions
     CfgUserActionDef(fza_ah64_wasGun, "WAS Gun", "Action the gun in your crew station");
     CfgUserActionDef(fza_ah64_wasRkt, "WAS Rockets", "Action the rockets in your crew station");
     CfgUserActionDef(fza_ah64_wasMsl, "WAS Missiles", "Action the missiles in your crew station");
-    CfgUserActionDef(fza_ah64_masterSafe, "Toggle Master Safe", "Arms or Safes the aircraft");
```

Comment:
Don't you need to add `armSafeButton`?

Reply:
Although it doesn't seem to be used anywhere, is armSafeButton even needed?

Reply:
Since we now have the arm/safe button as a clickable object, we no longer need the keybind